### ISSUE TITLE
Remote download folder files?

### ISSUE THREAD

**Issue by killmasta93** [Ylianst/MeshAgent#34]
Hi,
I was wondering if there will be a future update to download the folder files? As there is the option to download the files but not the option to download the whole folder?

Thank you

**Comment by killmasta93** [comment]
bump?

**Comment by krayon007** [comment]
Yeah, I added support for tarring files/folders a while ago, but we still need to add UX support into the webpage.

**Comment by killmasta93** [comment]
Thank you so much if you need to test it out before let me know

**Comment by killmasta93** [comment]
any updates on the WebGUI to download folders?
